src/utilities/utilities.js

```diff
@@ -749,3 +749,41 @@ export const met_typical_tasks = {
   Basketball: 6.3,
   Wrestling: 7.8,
 };
+
+/**
+ * Total Clothing insulation of typical ensembles
+ * @public
+ * @memberof reference_values
+ * @docname Typical ensembles insulation, [clo]
+ *
+ * @param {string} ensembles - Typical ensembles
```

Comment:
can you specify the strings that the function can take so users get autocomplete:
```javascript
{"Walking shorts, short-sleeve shirt" | "Typical summer indoor clothing"...}
```

Reply:
I'm not sure if I got it properly. I updated the @param part and listed all the valid ensembles, not sure if that's what you expected.

Reply:
yes, but you need to remove the `ensembles :` part